packages/client/hmi-client/src/components/models/Model.vue

```diff
@@ -644,4 +643,8 @@ const description = computed(() => highlightSearchTerms(model.value?.description
 	width: 100%;
 	height: 100%;
 }
+
```

Comment:
Consider moving `.constrain-width` and classes or any other repeated styles used among asset components to `asset.scss`. The styling with these components are quite similar so any shared styles belong there.